app/scripts/controllers/main.js

```diff
@@ -174,9 +180,7 @@ angular.module('moodCatApp')
         }
 
         //Send the mesage to the server
-        chatService.sendChatMessage($scope.chatMessage, $scope.room.id).then(function(res) {
-          $scope.messages.push(res);
-        });
+        chatService.sendChatMessage($scope.chatMessage, $scope.room.id);
```

Comment:
I changed this, but the fetching happens after the empty line is set.
Otherwise it won't empty the chatbox, because it's in the method.